data_summaries/AgentInstruct.json

```diff
@@ -0,0 +1,301 @@
+            "Instruction Following",
+            "Chain-of-Thought"
+        ],
+        "Text Sources": [
+            "crowdsourced"
+        ],
+        "Model Generated": [],
+        "Format": [
+            "Multi-turn Dialog"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [
+            "ALFRED"
+        ],
+        "Creators": [
+            "University of Washington",
+            "Microsoft",
+            "Carnegie Mellon University"
+        ],
+        "Licenses": [
+            {
+                "License": "MIT License",
+                "License URL": "https://github.com/alfworld/alfworld"
+            }
+        ],
+        "License Notes": "The used ALFRED dataset is licensed under the MIT License https://github.com/askforalfred/alfred/tree/master and the used TextWorld is licensed under the GNU General Public License (GPL) v2.0 License. The authors only state that ALFworld is licensed under the MIT License.",
+        "License Verified By": "Christopher",
+        "Dataset Filter IDs": [
+            "alfworld"
+        ]
+    },
+    "AgentInstruct-webshop": {
+        "Unique Dataset Identifier": "AgentInstruct-webshop",
+        "Dataset Name": "Webshop",
+        "Paper Title": "WebShop: Towards Scalable Real-World Web Interaction with Grounded Language Agents",
+        "Dataset URL": "https://huggingface.co/datasets/THUDM/AgentInstruct/viewer/default/webshop",
+        "GitHub URL": "https://github.com/princeton-nlp/WebShop",
+        "Hugging Face URL": "https://huggingface.co/datasets/THUDM/AgentInstruct/viewer/default/webshop",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2207.01206",
+        "Semantic Scholar Corpus ID": 250264533,
+        "Collection": "AgentInstruct",
+        "Collection URL": "https://huggingface.co/datasets/THUDM/AgentInstruct",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Instruction Following",
+            "Chain-of-Thought"
+        ],
+        "Text Sources": [
+            "crowdsourced",
+            "amazon.com",
+            "web search queries"
+        ],
+        "Model Generated": [],
+        "Format": [
+            "Multi-turn Dialog"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [],
+        "Creators": [
+            "Princeton University"
+        ],
+        "Licenses": [
+            {
+                "License": "MIT License",
+                "License URL": "https://github.com/princeton-nlp/WebShop/blob/master/LICENSE.md"
+            }
+        ],
+        "License Notes": "The authors collected data on various products from Amazon.com and retrieved text and crowdsourced instructions from annotators. It is possible that some of these annotations have additional licenses. Therefore, the license pertains to the framework that was used to create the dataset and is mentioned by the authors.",
```

Comment:
If I understand correctly, webshop is not a dataset but a tool used to create the dataset?

Reply:
it is a combination of real Amazon products and crowd-sourced text instructions, used by WebShop for the simulation.